torch/csrc/jit/script/module.cpp

```diff
@@ -368,6 +369,37 @@ void Module::dump(
             << std::endl;
 }
 
+namespace {
+void export_opnames(const script::Module& m, std::unordered_set<std::string>& opnames) {
+  auto methods = m.get_methods();
+  for (const auto& method : methods) {
+    const auto& func = method.function();
+    torch::jit::Code code(func.graph());
```

Comment:
Why do we need to construct the `Code` object? Will traversing all nodes in the graph not be enough for some reason?

Reply:
I think the code would contain the final ops for interpreter (when we bypass the optimizations in mobile). I'm not sure if emitting nodes would introduce more ops in future? Another reason is that I'm not sure how to know if a node is an operator node.

Reply:
All exporters use nodes directly and hence this function should use that too. Also, `Code` doesn't do any optimizations, it just wraps some nodes with Instructions - it's an internal detail of how interpreter works and should not affect exporting.